### PR TITLE
chore: Refactor server initialization and add router plugin

### PR DESCRIPTION
**Pull request by BarryGeeDefra**
Not ready to merge.

### CODE REVIEW

File: app/llm/ai.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ const embeddings = () => {
 }
 
 const model = (temperature = 0.9) => {
+  console.log(aiConfig)
```
Comment (alistairknight) [inline app/llm/ai.js]: Can you remove the console log

File: app/config/ai.js [hunk 2]
```diff
@@ -1,7 +1,9 @@
+require('dotenv').config()
+
 const aiConfig = {
   azureOpenAIApiInstanceName: process.env.OAI_INSTANCE_NAME,
   azureOpenAIApiKey: process.env.OAI_API_KEY,
-  azureOpenAIApiVersion: process.env.OAI_API_VERSION
+  azureOpenAIApiVersion: process.env.OAI_API_VERSION,
```
Comment (alistairknight) [inline app/config/ai.js]: Can you remove the last comma

File: app/llm/ai.js (hunk 1 above)
Comment (BarryGeeDefra) [reply app/llm/ai.js]: of course. knew id miss one!

File: app/config/ai.js (hunk 2 above)
Comment (BarryGeeDefra) [reply app/config/ai.js]: done.

**Review by alistairknight — APPROVED** [review]

**Pull request merged**